plugins/nodemon/src/tasks/nodemon.ts

```diff
@@ -42,7 +42,7 @@ export default class Nodemon extends Task<typeof NodemonSchema> {
       PORT: port.toString(),
       ...process.env
     }
-    nodemon({ script: entry, env, stdout: false })
+    nodemon({ script: entry, env, stdout: false, configFile: configPath })
```

Comment:
nitpick: configPath can potentially be undefined here because we don't have a defaultOption for it. If this is causing nodemon to log undefined I think we should handle the case when configPath is undefined

Reply:
Where is `configPath` (or `configFile`) logged?

The default option for `configFile` is `undefined` as we want it to look for the config file in the default search paths (`$PWD` and `$HOME`) if `configPath` isn't set in Tool Kit's config.

Reply:
This is more like calling `nodemon --config example.json server/app.js`, which does not affect the `node` invocation that is logged. Setting the `configFile` argument to `undefined` is equivalent to not setting it at all, causing `nodemon` to look up the config file in its default search paths instead.

Reply:
ah fair enough, if setting the configFile argument to undefined in the api won't reflect as undefined in the logs then that's fine